src/conventions/h-group/urgent-actions.js

```diff
@@ -313,7 +313,10 @@ export function find_urgent_actions(game, play_clues, save_clues, fix_clues, sta
 
 				const chop = common.chop(state.hands[state.ourPlayerIndex]);
 
-				if (state.clue_tokens === 0 && chop !== undefined) {
+				// As a last resort, only scream discard if it is critical.
+				const save_card = game.players[target].chop(state.hands[target]);
+				const remaining = cardCount(state.variant, save_card) - state.discard_stacks[save_card.suitIndex][save_card.rank - 1];
```

Comment:
Can't this be replaced with just `state.isCritical(save_card)`?

Reply:
Done, thanks!